test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -177,17 +222,14 @@ public void testInitOfUrlMetadataListerDocIdIsUrlTrue() throws Exception {
 
   private Config createStandardConfig(Map<String, String> moreEntries) {
     Map<String, String> configEntries = new HashMap<String, String>();
-    // driverClass must be specified, but (other than it pointing at a valid
-    // class), the value does not matther.
-    configEntries.put("db.driverClass",
-        "com.google.enterprise.adaptor.database.DatabaseAdaptor");
-    configEntries.put("db.url", "must be set");
-    configEntries.put("db.user", "must be set");
-    configEntries.put("db.password", "must be set");
+    configEntries.put("db.driverClass", JdbcFixture.DRIVER_CLASS);
+    configEntries.put("db.url", JdbcFixture.URL);
+    configEntries.put("db.user", JdbcFixture.USER);
+    configEntries.put("db.password", JdbcFixture.PASSWORD);
     configEntries.put("db.uniqueKey", "must be set:string");
-    configEntries.put("db.everyDocIdSql", "must be set");
+    configEntries.put("db.everyDocIdSql", "select 42 as \"must be set\"");
```

Comment:
I suggest we compromise on our two approaches. Set all these (must be set)  to empty string if that will cause them to pass your validation.  Any test will then need to set uniqueId, queries, etc as needed to function.  That also avoids setting them to empty string in all the tests where you added it.